build/crds/build-crds.sh

```diff
@@ -31,11 +31,11 @@ CRDS_BEFORE_1_16_FILE_PATH="${SCRIPT_ROOT}/cluster/examples/kubernetes/ceph/pre-
 #############
 # FUNCTIONS #
 #############
-# ensures the vendor dir has the right deps, e,g. volume replication controller
-if [ ! -d vendor/github.com/csi-addons/volume-replication-operator/api/v1alpha1 ];then
-  echo "Vendoring project"
-  go mod vendor
-fi
+
+generate_vol_rep_crds() {
+  echo "Generating volume replication crds in crds.yaml"
+  "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="github.com/csi-addons/volume-replication-operator/api/v1alpha1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
```

Comment:
If we remove this, I don't understand why we cannot remove `crds.go`. Or is it a leftover?